fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1 +1 @@
-<p>This Android app allows you your phone to present itself to a connected device as a keyboard and mouse. Since this happens at a very low level, the connected device sees it exactly as a normal keyboard and mouse, meaning you don't need to install any software on the connected computer! For this reason, it will even work if the connected device is in its BIOS/UEFI. Even multimedia keys are supported, meaning you can send volume up/down keys to the connected computer to control media.</p><p>Bonus description for nerds: This app creates (and provides a simple UI for you to communicate with) a USB HID gadget consisting of a keyboard and mouse character device (<code>/dev/hidg0</code> and <code>/dev/hidg1</code> respectively). If you wanted to, you could just use this app to create the character device, then communicate solely through some C program, shell script, or whatever else you wanted as long as it had the right permissions to access the character devices. Beware of SELinux policies. If anyone is actually doing that, let me know, I'd be glad to help if necessary.</p><p>For details on setup and usage, please consult the project's Readme.</p>
\ No newline at end of file
+<p>This Android app allows you your phone to present itself to a connected device as a keyboard and mouse. Since this happens at a very low level, the connected device sees it exactly as a normal keyboard and mouse, meaning you don't need to install any software on the connected computer! For this reason, it will even work if the connected device is in its BIOS/UEFI. Even multimedia keys are supported, meaning you can send volume up/downkeys to the connected computer to control media. <b>I strongly recommend using [Hacker's Keyboard](https://github.com/klausw/hackerskeyboard) to use special keys such as modifier and function keys)</b>, but most functionality will work with any standard Android keyboard.</p><p>Bonus description for nerds: This app adds (and provides a simple UI for you to communicate with) two character devices (<code>/dev/hidg0</code> and <code>/dev/hidg1</code> respectively) by adding HID functions to the default Android USB gadget. If you wanted to, you could just use this app to create the character devices, then communicate solely through some C program, shell script, or whatever else you wanted as long as it had the right permissions to access the character devices and knew how to send proper reports to them. Beware of SELinux policies if you do that though. If anyone is actually doing that, let me know, I'd be glad to help if necessary.</p><p>For more details on setup and usage, please consult the project's Readme.</p>
```

Comment:
Careful with your links there. Better not embed Markdown links (`[Hacker's Keyboard](https://github.com/klausw/hackerskeyboard)`) but stick to HTML.

That said: Hacker's Keyboard seems to be unmaintained. There are some alternatives providing the same functionality you refer to here. See e.g. [Unexpected-Keyboard](https://github.com/Julow/Unexpected-Keyboard).

Reply:
thanks! I copy pasted from the README and missed that. I was also searching for an alternative to Hacker's Keyboard, so that link is super helpful!!